lib/serial_terminal.pm

```diff
@@ -232,4 +236,37 @@ sub upload_file {
       or die("Failed to finally copy file from '$tmpfilename' to '$dst'");
 }
 
+
+sub reboot {
+    my (%args) = @_;
+    $args{check} //= 1;
+    $args{console} //= testapi::current_console;
+    $args{reboot_cmd} //= 'systemctl reboot';
+    $args{timeout} //= 300;
+    $args{minimum_boot_time} = 3;
+
+    bmwqemu::log_call(%args);
+
+    my $uptime_start;
```

Comment:
it is just IMHO so feel free to ignore this , but `_start` confuses me here and looks unrelated :
```suggestion
    my $uptime;
```